client.go

```diff
@@ -72,9 +72,7 @@ type Marathon interface {
 	// get a list of tasks for a specific application
 	Tasks(application string) (*Tasks, error)
 	// get a list of all tasks
-	AllTasks() (*Tasks, error)
-	// get a listing of the task ids
-	ListTasks() ([]string, error)
+	AllTasks(status string) (*Tasks, error)
```

Comment:
Perhaps we could use url.Values (https://github.com/gambol99/go-marathon/blob/master/application.go#L265), the benefit being we don't have to change anything if or when the API adds another option ... what you reckon?